src/applications/personalization/dashboard-2/tests/e2e/appointments-error.cypress.spec.js

```diff
@@ -3,8 +3,8 @@ import { mockFeatureToggles } from './helpers';
 import { mockUser } from '@@profile/tests/fixtures/users/user.js';
 
 import ERROR_400 from '~/applications/personalization/dashboard-2/utils/mocks/ERROR_400.js';
+import ERROR_500 from '@@profile/tests/fixtures/500.json';
 import PARTIAL_ERROR from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_VA_APPOINTMENTS_PARTIAL_ERROR.js';
-import MOCK_FACILITIES from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_FACILITIES.json';
```

Comment:
We actually don't need to mock the facilities endpoint. The facilities endpoint is _only_ called if we successfully fetch the appointment data. We do have a single test to make sure we handle an _error_ when trying to get the facilities, but in this spec file there is never a need to mock a successful response from the facilities endpoint.